cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -36,4 +36,6 @@
   public static final String WORKFLOW_BATCH = "workflowBatch";
 
   public static final String SCHEDULE_NAME = "scheduleName";
+
+  public static final String CONCURRENT_PROGRAM_RUN = "concurrent.program.run";
```

Comment:
@chtyim yes. I agree. We can have it as a part of Workflow properties and only pass in as runtime arguments if need to be overridden.